en/app.ftl

```diff
@@ -321,12 +323,17 @@ premium-promo-perks-headline = Why upgrade to { -brand-name-firefox-relay-premiu
 premium-promo-perks-lead-2 = With { -brand-name-firefox-relay-premium }, you get all the inbox protection and management of { -brand-name-relay }, but with unlimited email masks and your own custom subdomain to make managing your inbox even easier.
 premium-promo-perks-cta-label = Upgrade now
 premium-promo-perks-cta-tooltip = Upgrade to { -brand-name-firefox-relay-premium }
+premium-promo-perks-pill-new = New!
 premium-promo-perks-perk-unlimited-headline-2 = Create unlimited email masks
 premium-promo-perks-perk-unlimited-body-2 = No more five-mask limit: with { -brand-name-relay-premium }, you can generate as many masks as you need to help protect your email inbox from spammers and online trackers. You can even reply to emails without exposing your true address.
 premium-promo-perks-perk-custom-domain-headline-2 = Choose your own custom subdomain
 premium-promo-perks-perk-custom-domain-body-2 = With a custom subdomain, you can create masks that are easier than ever to remember and share. Need one for restaurant reservations? Use one like food@mydomain.mozmail.com — No need to create the mask beforehand.
 premium-promo-perks-perk-dashboard-headline-2 = Control your masks from the dashboard
 premium-promo-perks-perk-dashboard-body-2 = Manage all your email masks in the easy-to-use dashboard: if you find that one receives unwanted messages, you can block those messages from reaching your inbox.
+premium-promo-perks-perk-block-promotionals-headline = Block promotional emails
+premium-promo-perks-perk-block-promotionals-body = With { -brand-name-relay-premium }, you can block promotional emails from reaching your inbox while still receiving emails like receipts or shipping information.
+premium-promo-perks-perk-tracker-blocking-headline = Remove email trackers
+premium-promo-perks-perk-tracker-blocking-body = Now { -brand-name-relay } can help you stop email tracking—your email masks will remove common email trackers from any emails forwarded to you, helping you stay invisible to trackers and advertisers.
```

Comment:
Should there be spaces before and after the hyphen?

Reply:
I'll double-check.

Reply:
I quote:

> yes, good catch, thank you

Fixed now!